website/docs/reference/node-selection/syntax.md

```diff
@@ -80,3 +80,113 @@ Note that when you're using `--selector`, most other flags (namely `--select` an
+- [The `state:` selector](/reference/node-selection/methods#the-state-method), whereby dbt can identify resources that are new or modified
+by comparing code in the current project against the state manifest.
+- [Deferring](/reference/node-selection/defer) to another environment, whereby dbt can identify upstream, unselected resources that don't exist in your current environment and instead "defer" their references to the environment provided by the state manifest.
+
+Together, these two features enable ["slim CI"](/guides/legacy/best-practices#run-only-modified-models-to-test-changes-slim-ci). We expect to add more features in future releases that can leverage artifacts passed to the `--state` flag.
+
+### Establishing state
+
+State and defer can be set by environment variables as well as CLI flags:
+
+<VersionBlock lastVersion="1.4">
+
+- `--state` or `DBT_ARTIFACT_STATE_PATH`: file path
+- `--defer` or `DBT_DEFER_TO_STATE`: boolean
+
+</VersionBlock>
+
+<VersionBlock firstVersion="1.5">
+
+- `--state` or `DBT_STATE`: file path
+- `--defer` or `DBT_DEFER`: boolean
+
+:::warning Syntax deprecated
+
+In dbt v1.5, we deprecated the original syntax for state (`DBT_ARTIFACT_STATE_PATH`) and defer (`DBT_DEFER_TO_STATE`). Although dbt supports backward compatibility with the old syntax, we will remove it in a future release that we have not yet determined.
+
+:::
+
+</VersionBlock>
+
+If both the flag and env var are provided, the flag takes precedence.
+
+#### Notes:
+- The `--state` artifacts must be of schema versions that are compatible with the currently running dbt version.
+- The path to state artifacts can be set via the `--state` flag or `DBT_ARTIFACT_STATE_PATH` environment variable. If both the flag and env var are provided, the flag takes precedence.
+- These are powerful, complex features. Read about [known caveats and limitations](/reference/node-selection/state-comparison-caveats) to state comparison.
+
+### The "result" status
+
+Another element of job state is the `result` of a prior dbt invocation. After executing a `dbt run`, for example, dbt creates the `run_results.json` artifact which contains execution times and success / error status for dbt models. You can read more about `run_results.json` on the ['run results'](/reference/artifacts/run-results-json) page. 
+
+The following dbt commands produce `run_results.json` artifacts whose results can be referenced in subsequent dbt invocations:  
+- `dbt run`
+- `dbt test`
+- `dbt build` (new in dbt version v0.21.0)
+- `dbt seed` 
+
+After issuing one of the above commands, you can reference the results by adding a selector to a subsequent command as follows: 
+
+```bash
+# You can also set the DBT_ARTIFACT_STATE_PATH environment variable instead of the --state flag.
+$ dbt run --select result:<status> --defer --state path/to/prod/artifacts
+```
+
+The available options depend on the node type: 
```

Comment:
This is the first time we mention "node" in this section with "About node" in the title. Can we add just a line that connects node to state? Is node selection running only specific resources in a given invocation of dbt, and the given invocation is the state?

Reply:
good flag! line 20 has an info callout that states:

> We use the terms "nodes" and "resources" interchangeably. These encompass all the models, tests, sources, seeds, snapshots, exposures, and analyses in your project. They are the objects that make up dbt's DAG (directed acyclic graph).

which is pretty far up the page. i added a couple of "reminders" in this section